src/TroykaOLED.h

```diff
@@ -1,176 +1,73 @@
-// установить тип аппаратной конфигурации COM выводов
-// за данной командой должен быть отправлен один байт, у которого:
-// четвёртый бит: 0-последовательная / 1-альтернативная 
-// пятый бит: 0-отключить COM Left/Right remap / 1-включить COM Left/Right remap
-#define SSD1306_SET_COM_PINS                    0xDA
-
-// установить частоту обновления дисплея
-// за данной командой должен быть отправлен один байт, старшие четыре бита которого определяют частоту, а младшие делитель
-#define SSD1306_SET_DISPLAY_CLOCK               0xD5
-// установить фазы DC/DC преобразователя
-// за данной командой должен быть отправлен один байт, старшие четыре бита которого определяют период, а младшие период
-#define SSD1306_SET_PRECHARGE_PERIOD            0xD9
-// установить VcomH (влияет на яркость)
-// за данной командой должен быть отправлен один байт, старшие четыре бита которого определяют напряжение
-// пример: 0000 - VcomH=0.65Vcc, 0010 - VcomH=0.77Vcc, 0011 - VcomH=0.83Vcc и т.д.
-#define SSD1306_SET_VCOM_DESELECT               0xDB
-// управление DC/DC преобразователем
-// за данной командой должен быть отправлен один байт:
-// 0x10 - отключить (VCC подается извне), 0x14 - запустить внутренний DC/DC
-#define SSD1306_CHARGE_DCDC_PUMP                0x8D
-
-// положение бита DC в командном байте
-// этот бит указывает что следующим байтом будет: 0-команда или 1-данные
-#define SSD1306_SHIFT_DC                        0x06
-// положение бита CO в командном байте
-// этот бит указывает что после следующего байта (команды или данных) будет следовать (если будет): 0-байт данных или 1-новый командный байт
-#define SSD1306_SHIFT_CO                        0x07
-// (CO=0, DC=0) => 0x00 контрольный байт после которого следует байт команды
-#define SSD1306_COMMAND                         (0 << SSD1306_SHIFT_CO) | (0 << SSD1306_SHIFT_DC)
-// (CO=0, DC=1) => 0x40 контрольный байт после которого следует байт данных
-#define SSD1306_DATA                            (0 << SSD1306_SHIFT_CO) | (1 << SSD1306_SHIFT_DC)
-
-#define OLED_TOP                                250
-#define OLED_BOTTOM                             251
-#define OLED_LEFT                               252
-#define OLED_RIGHT                              253
-#define OLED_CENTER                             254
-#define OLED_THIS                               255
-
-#define IMG_RAM                                 0
-#define IMG_ROM                                 1
-
-#define TXT_CP866                               0
-#define TXT_UTF8                                1
-#define TXT_WIN1251                             2
-
-#define BLACK                                   0
-#define WHITE                                   1
-#define INVERSE                                 2
+constexpr int16_t OLED_TOP = 32762;
+constexpr int16_t OLED_BOTTOM = 32763;
+constexpr int16_t OLED_LEFT = 32764;
+constexpr int16_t OLED_RIGHT = 32765;
+constexpr int16_t OLED_CENTER = 32766;
+constexpr int16_t OLED_THIS = 32767;
```

Comment:
Not. This const`s are using as well as fixed position values.  Using enum value as int will confuse the compiler. It will generate unnecessary warnings.